### PR TITLE
Views

### PR DESCRIPTION
**Pull request by dieum512**
In this milestone I:

- [x] Added data in the database
- [x] Used preprocessed HTML file with embedded Ruby code.
- [x] Used layouts and templates for shared content.
- [x] Fixed some linter errors

Thank you for the time to correct this project :wink:

### CODE REVIEW

File: app/views/users/index.html.erb
Comment (franclobo) [inline app/views/users/index.html.erb:1]: - [ ] Kindly add the root route to `user#index` page for a better UX.

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1 +1,35 @@
...
+    <div class="bio">
+        <h3>Bio</h3>
+        <p>
+            <%= @user.bio %>
+        </p>
+    </div>
+    <div class="recent-posts">
+        <% @user.recent_posts.each do |post| %>
+            <div class="user-posts">
+                <h3><%= link_to post.title, user_post_path(post) %></h3>
```
Comment (franclobo) [inline app/views/users/show.html.erb:24]: - [ ] Everything is fine when I open the user#show page, but when I select a post to see the post#show page it returns the wrong path, it is users/post_id/posts/user_id it should be users/user_id/posts/post_id, please correct the path to open the correct page 👇🏽 

https://github.com/dieum512/Blogg-app-2/assets/58642949/80a785be-a314-4325-bd21-1a7761994940

File: app/views/posts/index.html.erb
Comment (franclobo) [inline app/views/posts/index.html.erb:1]: - [ ] When I clicked on the See all posts button it araised all the posts that I have created, it should be just the post from the current user. Please 🙏🏽 correct this issue for a better UX 👇🏽 

https://github.com/dieum512/Blogg-app-2/assets/58642949/4c27e5eb-0ebf-4b1e-8686-644d441ec894

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @dieum512, 

# Required Changes ♻️

so-close-jim-halpert

Good job so far!
You are doing it well 👏🏽 there are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- No linter errors ✔️ 
- Descriptive PR ✔️ 
- Correct Git flow ✔️ 
- Use preprocessed HTML file with embedded Ruby code. ✔️ 
- Use layouts and templates for shared content. ✔️ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- **Note:** If you would like to create a lot of users, posts, and comments. We could install gem 'faker' and create them on seed.rb file as follows 👇🏽 

```ruby
require "faker"

# Create Users
10.times do
  User.create!(
    name: Faker::Name.name,
    photo: Faker::Avatar.image,
    bio: Faker::Lorem.paragraph(sentence_count: 2),
    posts_counter: 0
  )
end

# Create Posts
50.times do
  Post.create!(
    title: Faker::Lorem.sentence(word_count: 3),
    text: Faker::Lorem.paragraph(sentence_count: 3),
    comments_counter: 0,
    likes_counter: 0,
    author_id: rand(1..10)
  )
end

# Create Comments
100.times do
  Comment.create!(
    text: Faker::Lorem.paragraph(sentence_count: 2),
    author_id: rand(1..10),
    post_id: rand(1..50)
  )
end
```

Finally, run in your terminal `rails db:drop db:create db:migrate db:seed` and `rails s`

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by divinecharlotte — APPROVED** [review]
# Hello @dieum512       😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say  👍🏻 ✅ 
### To Highlight 💯


- Good implementation of the project requirements👍 
- well-organized code structure✔ 👍
 




## Congratulations! 🎉 



Cheers, and Happy coding!👏👏👏


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**